napalm_logs/server.py

```diff
@@ -238,6 +275,13 @@ def start(self):
                     'model_name': 'unknown'
                 }
                 self.publisher_pub.send(umsgpack.packb(to_publish))
+                napalm_logs_server_messages_as_is_queued.inc()
```

Comment:
I think maybe `napalm_logs_server_messages_unknown_queued` might be a little clearer.